qutebrowser/browser/greasemonkey.py

```diff
@@ -146,15 +157,27 @@ class GreasemonkeyManager(QObject):
 
     def __init__(self, parent=None):
         super().__init__(parent)
+        self._run_start = []
+        self._run_end = []
+        self._run_idle = []
+        self._in_progress_dls = []
+        self._pending_dls = []
+
         self.load_scripts()
+        app = objreg.get('app')
```

Comment:
I noticed you said to use `QApplication.instance()` on another PR but I must protest. I know python is ... flexibly typed and all but that implies it is returning a QApplication singleton and QApplication doesn't have a `new_window` signal.

Reply:
Fair point. I'm trying to get rid of `objreg` in the long run, but for places which actually need the subclassed `Application`, using `objreg` is probably okay (for now :wink:).